public/index.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html lang='en'>
+  <head>
+    <title>Instructor Center</title>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>
+    <link rel='apple-touch-icon' sizes='180x180' href='/apple-touch-icon.png'>
+    <link rel='icon' type='image/png' href='/favicon-32x32.png' sizes='32x32'>
+    <link rel='icon' type='image/png' href='/favicon-16x16.png' sizes='16x16'>
+    <link rel='manifest' href='/manifest.json'>
+    <link rel='mask-icon' href='/safari-pinned-tab.svg' color='#555555'>
+    <meta name='theme-color' content='#ffffff'>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+      ga('create', 'UA-36512724-2', 'auto');
+      ga('send', 'pageview');
+    </script>
```

Comment:
Using an inline google analytics script for now; would like to move this inside a module with an npm package but was running into issues. Have on my list to investigate more.